randovania/exporter/patch_data_factory.py

```diff
@@ -88,8 +88,12 @@ def create_default_patcher_data_meta(self) -> PatcherDataMeta:
 
     def _attach_to_sentry(self) -> None:
         with sentry_sdk.isolation_scope() as scope:
+            description_json = self.description.original_dict
+            if description_json is None:
```

Comment:
Does the layout get saved by the server? At least I'm assuming something like that, as otherwise this would be always None I think.
Also does the layout for multis get created based off of a json dict?